book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] We need both   _Getters_ and _Setters_ for _@rentals_ to be present, therefore kindly ensure to mark _rentals_ with _attr_accessor_, similar to:
```ruby
  attr_accessor :rentals